extensions/wikibase/src/org/openrefine/wikibase/editing/EditBatchProcessor.java

```diff
@@ -183,14 +185,22 @@ public EditResult performEdit()
                 // New entities
                 ReconEntityIdValue newCell = (ReconEntityIdValue) update.getEntityId();
                 EntityIdValue createdDocId;
+                String label = "";
```

Comment:
Maybe this variable could be called `name` instead, because it's going to be stored as the reconciliation candidate's name. It's not always a label (since for MediaInfo entities it is a filename).